doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -0,0 +1,13 @@
+API Changelog 
+=============
+
+.. contents:: |toctitle|
+    :local:
+    :depth: 1
+
+6.0.0
```

Comment:
There is no 6.0.0. Do you mean 6.0?

Reply:
Fixed it, was just leaving it like that in case there is a smaller release at some point. 

Thanks!

Reply:
But that's not what the release is called. Can you please change it? Also, in which version did the change happen? I thought this was unknown. "sometime between 5.9 and 6.0", right (from the issue)?

Reply:
Changed to 6.0 in e0350e7. Thanks.